src/charging-station/ChargingStation.ts

```diff
@@ -132,6 +140,8 @@ export class ChargingStation {
   private readonly sharedLRUCache: SharedLRUCache;
   private webSocketPingSetInterval!: NodeJS.Timeout;
   private readonly chargingStationWorkerBroadcastChannel: ChargingStationWorkerBroadcastChannel;
+  private reservations?: Reservation[];
```

Comment:
Reservations are linked to a connector, right? 
So they should be part of the ConnectorStatus data structure.

Reply:
Good point, I'll change that

Reply:
How OCPP 2.0 handle reservation? If it's at the EVSE level, then it should also be added to `EvseStatus` and helpers must behave accordingly (connectors or evses set in CS template) .

Reply:
The simulator code support evses or(exclusif) connectors in the template, so your reservation implementation must support both configurations, independently of the OCPP stack version.